catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/TemplateTransformer.java

```diff
@@ -99,11 +128,30 @@ public Metacard toQueryTemplateMetacard(Map<String, Object> formTemplate) {
       if (querySettings != null) {
         metacard.setQuerySettings(querySettings);
       }
+
+      Result result = metacardAlreadyExists(id, filterBuilder, catalogFramework);
+
+      // Perform update of old Metacard if already exists
+      if (result != null) {
+        Metacard oldMetacard = null;
+        oldMetacard = result.getMetacard();
+        oldMetacard.setAttribute(new AttributeImpl(Core.MODIFIED, new Date()));
+        oldMetacard.setAttribute(new AttributeImpl(Core.TITLE, title));
+        oldMetacard.setAttribute(new AttributeImpl(Core.DESCRIPTION, description));
+        oldMetacard.setAttribute(new AttributeImpl(QUERY_TEMPLATE_FILTER, filterXml));
```

Comment:
❓ Couldn't there potentially be other values on the new metacard that aren't updated in this approach? (i.e. `sorts`, `sources`, `detail-level`, and various other things on `QueryMetacardTypeImpl`)?

Reply:
I think what you have here + `sorts`, `sources`, `federation`, `enterprise`, `detail-level` would be everything that could potentially be changed when updating/saving a search form